nms/app/packages/magmalte/server/apicontroller/auditLoggingDecorator.js

```diff
@@ -0,0 +1,191 @@
+/*
```

Comment:
This file seems unrelated to the PR? It seems like this is just being copied from js-core, so it's fine here but in the future that would be better as a separate PR.

Reply:
I had copied apiController/routes over because i had to change something here for this PR. Specifically the change mentioned below
-const networkTypeRegex = '(cwf|feg|lte|symphony|wifi)';
+const networkTypeRegex = '(cwf|feg|lte|feg_lte|symphony|wifi)';

However the routes depends on auditLoggingDecorator, so I bought in that as well and just moved all of the Magma specific code from platform-server to here.